puppet/zulip/files/logrotate/zulip

```diff
@@ -1,8 +1,13 @@
-/var/log/zulip/server.log /var/log/zulip/workers.log /var/log/zulip/manage.log {
+/var/log/zulip/*.log {
```

Comment:
Yeah, I think the problem is that supervisord rotates logs all nicely without your needing to restart anything, however, it doesn't compress them, which is really stupid.

There's a pretty disappointing long-open issue about it: https://github.com/Supervisor/supervisor/issues/322.

My theory is that since we don't need a lot of history in the `supervisord` log files, we can just set a pretty low limit for how much is kept.